migrations/versions/2019-02-12T194215.py

```diff
@@ -0,0 +1,274 @@
+"""empty message
+
+Revision ID: 9cea5090cefb
+Revises: fb887393e975
+Create Date: 2019-02-12 19:42:15.920435+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '9cea5090cefb'
+down_revision = 'fb887393e975'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('companies',
+    sa.Column('created_on', sa.DateTime(), nullable=False),
+    sa.Column('updated_on', sa.DateTime(), nullable=False),
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('code', sa.String(), nullable=False),
+    sa.Column('address', sa.String(), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('code'),
+    sa.UniqueConstraint('name')
+    )
+    op.create_table('customers',
```

Comment:
@vatavuk why do we have this one? Is it mistake?

Reply:
@vladarefiev I will remove migrations from PR, we don't need them, I've added them by mistake